src/main/models/subscription.ts

```diff
@@ -0,0 +1,12 @@
+export interface Subscription {
+
+  id: string;
+  userId: string;
+  searchType: string;
+  searchValue: string;
+  channel: string;
+  createdDate: any;
```

Comment:
these attr. arent returned in the subscription object, its just the case subscriptions and court subscriptions, this object is a `UserSubscriptions` object which contain mutliple subs, each one contains created date and id inside the subscriptions 1 to apply to all

Reply:
deleted